### PR TITLE
stargazers beware

### PR DESCRIPTION
**Pull request by emanresusername**
resisting powerful urge to @mention all the stargazers troll

### CODE REVIEW

**Comment by timss** [comment]
It got deleted, here's a couple of snapshots instead if they are to be included:

- https://archive.fo/DNf2M
- https://webcache.googleusercontent.com/search?q=cache:https://github.com/jdm-contrib/justdelete.me/issues/86

**Comment by emanresusername** [comment]
full disclosure for anybody who follows through to here, [they did apologize](https://github.com/jdm-contrib/justdelete.me/issues/88)

**Comment by timss** [comment]
Nice of you to include their apologies :+1: 

Markdown nitpick; I think it'd be better to keep the original hyperlink, but include mirrors like so:

```markdown
https://github.com/jdm-contrib/justdelete.me/issues/86 [mirrors: [1](https://webcache.googleusercontent.com/search?q=cache:https://github.com/jdm-contrib/justdelete.me/issues/86), [2](https://archive.fo/DNf2M)]
```

https://github.com/jdm-contrib/justdelete.me/issues/86 [mirrors: [1](https://webcache.googleusercontent.com/search?q=cache:https://github.com/jdm-contrib/justdelete.me/issues/86), [2](https://archive.fo/DNf2M)]

**Comment by jomo** [comment]
Just a quick note, linking webcache.googleusercontent.com is a bit pointless because it is (as the name implies) a cache, not a web archive. The content won't stay there for long.

**Comment by emanresusername** [comment]
good point... i'm biased towards whatever's the least work for me (i.e. do nothing :sweat_smile:) but glad to to change however people think is best

File: README.md [hunk 1]
```diff
@@ -2,81 +2,82 @@
...
+[https://github.com/strongloop/loopback/issues/1079](https://github.com/strongloop/loopback/issues/1079)
 
-https://github.com/tc39/proposal-cancelable-promises/issues/70
+[https://github.com/tc39/proposal-cancelable-promises/issues/70](https://github.com/tc39/proposal-cancelable-promises/issues/70)
 
-https://github.com/twitter/scrooge/issues/222
+[https://github.com/twitter/scrooge/issues/222](https://github.com/twitter/scrooge/issues/222)
 
-https://github.com/zenparsing/es-observable/pull/97
+[https://github.com/zenparsing/es-observable/pull/97](https://github.com/zenparsing/es-observable/pull/97)
```
Comment (nikolas) [inline README.md]: Why all these link updates? they're needlessly complicating this document

File: README.md [hunk 2]
```diff
@@ -2,81 +2,82 @@
 
 This is a curated collection of "dramatic" github interactions.
 
+[https://github.com/jdm-contrib/justdelete.me/issues/86](https://github.com/jdm-contrib/justdelete.me/issues/86) [mirrors: [1](https://webcache.googleusercontent.com/search?q=cache:https://github.com/jdm-contrib/justdelete.me/issues/86), [2](https://archive.fo/DNf2M)]
```
Comment (nikolas) [inline README.md]: use something like archive.org rather than google web cache if it's no longer on github.

File: README.md (hunk 1 above)
Comment (emanresusername) [inline README.md]: interesting just tried it from a different computer and the links were automatically clickable, but from the one i made this commit from the links were not without the markdown link syntax 😕 
notsureif does github guarantee http links should be clickable and it was just my other browsers config fault? or have other people experienced the non-clickable links case?

**Comment by emanresusername** [comment]
ok, couldn't reproduce the non-linking thing when i got back home, so reverted that, and took out cache link

**Pull request merged**